marketplace/wpp_templates/models.py

```diff
@@ -0,0 +1,119 @@
+import uuid
+
+from django.contrib.auth import get_user_model
+from django.utils import timezone
+from django.db import models
+
+from marketplace.applications.models import App
+
+User = get_user_model()
+
+
+class TemplateMessage(models.Model):
+    CATEGORY_CHOICES = (
+        ("ACCOUNT_UPDATE", "WhatsApp.data.templates.category.account_update"),
+        ("PAYMENT_UPDATE", "WhatsApp.data.templates.category.payment_update"),
+        ("PERSONAL_FINANCE_UPDATE", "WhatsApp.data.templates.category.personal_finance_update"),
+        ("SHIPPING_UPDATE", "WhatsApp.data.templates.category.shipping_update"),
+        ("RESERVATION_UPDATE", "WhatsApp.data.templates.category.reservation_update"),
+        ("ISSUE_RESOLUTION", "WhatsApp.data.templates.category.issue_resolution"),
+        ("APPOINTMENT_UPDATE", "WhatsApp.data.templates.category.appointment_update"),
+        ("TRANSPORTATION_UPDATE", "WhatsApp.data.templates.category.transportation_update"),
+        ("TICKET_UPDATE", "WhatsApp.data.templates.category.ticket_update"),
+        ("ALERT_UPDATE", "WhatsApp.data.templates.category.alert_update"),
+        ("AUTO_REPLY", "WhatsApp.data.templates.category.auto_reply"),
+        ("TRANSACTIONAL", "WhatsApp.data.templates.category.transactional"),
+        ("MARKETING", "WhatsApp.data.templates.category.marketing"),
+        ("OTP", "WhatsApp.data.templates.category.otp"),
+    )
+
+    TEMPLATE_TYPES_CHOICES = (
+        ("MEDIA", "WhatsApp.data.templates.type.media"),
+        ("INTERACTIVE", "WhatsApp.data.templates.type.interactive"),
+        ("TEXT", "WhatsApp.data.templates.type.text"),
+    )
+
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+
```

Comment:
Excess blank lines at: 37, 39, 41, 43, 45, 47. 
Extra blank lines can be used (sparingly) to separate groups of related functions.
More details in: https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Done!